recipes/mlpack/all/conanfile.py

```diff
@@ -113,12 +98,22 @@ def package_info(self):
         if self.settings.get_safe("compiler.libcxx") in ["libstdc++", "libstdc++11"]:
             self.cpp_info.system_libs.append("atomic")
 
-        self.cpp_info.cflags = self._openmp_flags
-        self.cpp_info.cxxflags = self._openmp_flags
-
+        # FIXME: Review this flags assignment. LLVM-OpenMP is already defining these ones.
+        #        Do we want to depend on llvm-openmp? Only for apple-clang?
+        #        For more info, see this comment https://github.com/conan-io/conan-center-index/pull/22353#issuecomment-2214593855
+        flags = []
+        # Based on https://github.com/Kitware/CMake/blob/v3.28.1/Modules/FindOpenMP.cmake#L104-L135
```

Comment:
Yes, this would be a much better solution and would not force the user to rely on `llvm-openmp` if it matters to them: 
- #22360